src/Client.php

```diff
@@ -19,13 +18,23 @@ class Client implements HttpClient
 {
     private $client;
     private $responseFactory;
-    private $streamFactory;
 
-    public function __construct(Artax\Client $client = null, ResponseFactory $responseFactory = null, StreamFactory $streamFactory = null)
-    {
+    /**
+     * @param Artax\Client    $client          HTTP client implementation.
+     * @param ResponseFactory $responseFactory Response factory to use or `null` to attempt auto-discovery.
+     * @param StreamFactory   $streamFactory   This parameter will be ignored and removed in the next major version.
```

Comment:
Yes, but that way we can tag `v0.1.1` and nobody even has to update their `composer.json`.